Packs/DarktraceEventCollector/Integrations/DarktraceEventCollector/DarktraceEventCollector.yml

```diff
@@ -0,0 +1,102 @@
+category: Analytics & SIEM
```

Comment:
not sure if this is the correct category or not, please make sure with Dima (in the XSOAR integration it's Network Security)